tests/test_modbus_rtu_block.py

```diff
@@ -104,23 +104,11 @@ def test_no_response(self, mock_client):
         self.assertFalse(len(self.last_notified[DEFAULT_TERMINAL]))
         blk.stop()
 
-    @patch("{}.sleep".format(ModbusRTU.__module__))
-    @patch('minimalmodbus.Instrument')
-    def test_execute_retry_forever(self, mock_client, mock_sleep):
-        ''' Test that retries will continue forever '''
```

Comment:
This is not the deisred behavior, and by dumb luck it was not being applied to actual configured blocks. Any new isntance of the block created in the designer ends up with the linear backpoff strat and 5 max retries anyway. Retries should be user-defined, not a static feature of the block. Especially in the case of these blocks, where the typical use case is a synchronous poll, so the request will be "retried" on the following interval anyway, we like to set max retries to 0.